contrib/librespot.service

```diff
@@ -9,7 +9,7 @@ User=nobody
 Group=audio
 Restart=always
 RestartSec=10
-ExecStart=/usr/bin/librespot -n "%p on %H"
+ExecStart=/bin/sh -c 'exec /usr/bin/librespot --name librespot@$(hostname -s)'
```

Comment:
This is mostly due to #200. I can wait for this to be merged until that issue has been resolved, if you'd rather not have this hack here.

I've tried with the iOS app too -- using a FQDN fails, and I tried things as basic as "`te.st`", and that fails. (if it works for you, let's please discuss that at #200).